### PR TITLE
feat: use lazy_static_include

### PR DESCRIPTION
**Pull request by KaiserKarel**
In debug mode, using `lazy_static_include!` saves on compilation time versus `include_bytes!`, as mentioned in #3996. I have refrained from changing  `include_bytes` in benches, as they are compiled in release mode anyway,

Test plan
---------
Current tests already cover this change. However we can no longer distribute debug mode binaries (I am not sure if this is currently done, but I highly doubt it).

### CODE REVIEW

**Comment by KaiserKarel** [comment]
`lazy_static_include` not checking if paths exist during tests/compilations did bite me in the butt a bit. At least I got familiar with the test framework. Let's see if the builds work now, else I will have a look next weekend.

**Comment by KaiserKarel** [comment]
I'm a bit too unfamiliar with what db-migration is testing, but I am suspecting that the build is spinning up a node in debug compilation?

**Review by bowenwang1996 — APPROVED** [review]

File: test-utils/testlib/Cargo.toml [hunk 1]
```diff
@@ -16,6 +16,7 @@ rand = "0.7"
 serde_json = "1"
 futures = "0.3"
 lazy_static = "1.4"
```
Comment (bowenwang1996) [inline test-utils/testlib/Cargo.toml:18]: is `lazy_static` still needed after this change?

**Comment by bowenwang1996** [comment]
> I'm a bit too unfamiliar with what db-migration is testing, but I am suspecting that the build is spinning up a node in debug compilation?

Can you try rebasing your change on top of master?

File: test-utils/testlib/Cargo.toml (hunk 1 above)
Comment (KaiserKarel) [reply test-utils/testlib/Cargo.toml:18]: Yeah, it's used in some places for test globals:

```
lazy_static! {
    static ref HEAVY_TESTS_LOCK: Mutex<()> = Mutex::new(());
}
```

**Review by ailisp — APPROVED** [review]

**Review by bowenwang1996 — APPROVED** [review]

**Pull request merged**